Script/item.dat

```diff
@@ -3844,4 +3843,3 @@ firstbornchild
     NameSingular = "daughter";
   }
 }
-
```

Comment:
Yeah I know, however that instance didn't add any significant noise in the diff or blame, so I decided it wasn't worth it to leave it alone :)